extensions/amp-stream-gallery/1.0/test/test-amp-stream-gallery.js

```
@@ -0,0 +1,180 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import '../../../amp-base-carousel/1.0/amp-base-carousel';
+import '../amp-stream-gallery';
+import {
+  createElementWithAttributes,
+  waitForChildPromise,
+} from '../../../../src/dom';
+import {setStyles} from '../../../../src/style';
+import {toArray} from '../../../../src/types';
+import {toggleExperiment} from '../../../../src/experiments';
+import {useStyles} from '../../../amp-base-carousel/1.0/base-carousel.jss';
+import {waitFor} from '../../../../testing/test-helper';
+
+describes.realWin(
+  'amp-stream-gallery',
+  {
+    amp: {
+      runtimeOn: true,
+      extensions: ['amp-stream-gallery:1.0'],
+    },
+  },
+  (env) => {
+    let win;
+    let element;
+    const userSuppliedChildren = [];
+
+    const styles = useStyles();
+
+    // ignore ResizeObserver loop limit exceeded
+    // this is ok in several scenarios according to
+    // https://github.com/WICG/resize-observer/issues/38
+    before(() => {
+      window.onerror = function (err) {
+        if (err === 'ResizeObserver loop limit exceeded') {
+          return false;
+        } else {
+          return err;
+        }
+      };
+    });
+
```

This is where the error is ignored @dvoytenko